src/consensus/pbft/libbyz/Replica.cpp

```diff
@@ -1075,6 +1101,18 @@ void Replica::emit_signature_on_next_pp(int64_t version)
   signed_version = version;
 }
 
+void Replica::activate_pbft_local_hooks()
+{
+  pbft_requests.set_local_hook(requests_local_hook);
+  pbft_pre_prepares.set_local_hook(pre_prepares_local_hook);
+}
+
+void Replica::deactivate_pbft_local_hooks()
```

Comment:
Unused?

Reply:
To be used in next PR where playback of ledger will happen via State messages